tox.ini

```diff
@@ -49,15 +48,14 @@ deps =
     cov: coverage
     cov: pytest-cov
 
-    numpy116: numpy==1.16.*
-    numpy117: numpy==1.17.*
-    numpy118: numpy==1.18.*
+    numpy125: numpy==1.25.*
+    numpy200: numpy==2.0.*
 
     astropy50: astropy==5.0.*
     astropy60: astropy==6.0.*
 
-    oldestdeps: numpy==1.19.*
-    oldestdeps: astropy==4.0.*
+    oldestdeps: numpy==1.25.*
+    oldestdeps: astropy==5.0.*
 
     devdeps: git+https://github.com/astropy/astropy.git#egg=astropy
```

Comment:
You can use nightly wheel here. Example: https://github.com/spacetelescope/synphot_refactor/blob/master/tox.ini

Reply:
Done.